app/assets/javascripts/components/features/ui/containers/status_list_container.jsx

```diff
@@ -27,6 +27,9 @@ const makeGetStatusIds = () => createSelector([
       if (showStatus) {
         const regex = new RegExp(columnSettings.getIn(['regex', 'body']).trim(), 'i');
         showStatus = !regex.test(statusForId.get('reblog') ? statuses.getIn([statusForId.get('reblog'), 'unescaped_content']) : statusForId.get('unescaped_content'));
+        if (showStatus && statusForId.get('spoiler_text').length > 0) {
+          showStatus = !regex.test(statusForId.get('reblog') ? statuses.getIn([statusForId.get('reblog'), 'unescaped_spoiler_content']) : statusForId.get('unescaped_spoiler_content'));
```

Comment:
considering that we're now checking against four possible content locations, maybe it's better to just append and then test them all in the same string?

something like:

```js
let content = statusForId.get('unescaped_content') || statusForId.get('reblog.unescaped_content');
content +=  || "";
content += statusForId.get('unescaped_spoiler_content') || statusForId.get('reblog.unescaped_content');
content += statusForId.get('unescaped_spoiler_content') || statusForId.get('unescaped_spoiler_content');

showStatus = !regex.test(content);
```